csharp/test/Apache.Arrow.Flight.TestWeb/Apache.Arrow.Flight.TestWeb.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Grpc" Version="2.33.1" />
```

Comment:
Not needed

```suggestion
```

Reply:
This is just for the tests where I use some of the extension methods, such as ToListAsync() etc, Is it considered bad practice to use it even in those cases?

Reply:
Removed it now, thanks!

Reply:
Grpc brings in some big (50+ meg) NuGet packages. I'd copy the source code to avoid brining them in.